MessageCenter/Classes/Client/SendBirdClient.swift

```diff
@@ -117,8 +117,29 @@ public class SendBirdClient: ClientProtocol {
     }
   
     public func disconnect(completion: @escaping () -> Void) {
+        if self.isConnected == false {
```

Comment:
How we are connecting to sendBird without disconnecting? the function implies that we should disconnect..

Reply:
@mohhamednabil I didn't understand the purpose of this PR, please elaborate..

Reply:
@ikarmakhan can you please explain the need for this ?